services/leecher/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "shotgrid-leecher"
-version = "0.3.2"
+version = "0.4.0"
```

Comment:
This is not related to this PR, but why is version changed in PR? Shouldn't we use standard development flow (having dev version in `develop` branch).

Reply:
yeah, good point, I will remove the version changes and rather add testing notes instruction for version bumping before testing.